sbt-dotty/src/dotty/tools/sbtplugin/DottyIDEPlugin.scala

```diff
@@ -15,6 +15,16 @@ import scala.collection.mutable.ListBuffer
 import DottyPlugin.autoImport._
 
 object DottyIDEPlugin extends AutoPlugin {
+  object autoImport {
+    val excludeFromIDE = settingKey[Boolean]("If true, do not import this project or configuration in the Dotty IDE")
```

Comment:
Not sure if it's a good idea, is it worth considering naming this key `ide-skip-project`? To match https://github.com/JetBrains/sbt-ide-settings/blob/750b993453fb3d1f31f371968d06e3fc792870a1/src/main/scala/sbtide/Keys.scala#L11-L12

Also, by convention, keys from the same plugin should ideally use the same prefix:
- ideLaunch
- ideExclude
- ideCommand
- ideRun

This convention makes it easy to explore available plugin keys from the sbt shell.

Reply:
>  is it worth considering naming this key ide-skip-project?

Using the same key name as another plugin seems like a bad idea in fact, because then they will conflict with each other, I have no idea what that does in the sbt repl but in build files you'll get errors due to ambiguity.

> Also, by convention, keys from the same plugin should ideally use the same prefix

Maybe yes, feel free to open an issue for that. I have no idea what the good practices for sbt plugins are :).

Reply:
Instead of defining a new key, can't we just use `skip in launchIDE := true`?

Reply:
That wouldn't be accurate since `launchIDE` just calls `configureIDE` and I don't think we can do `skip in configureIDE` since `configureIDE` is a command and not a task. It also wouldn't be complete since we also want to disable the config for `compileForIDE`. All in all, it seems better to have a specific key for this purpose :).